bootstrap/frontend/src/main/scala/io/udash/bootstrap/tooltip/TooltipUtils.scala

```diff
@@ -2,13 +2,31 @@ package io.udash.bootstrap
 package tooltip
 
 import com.avsystem.commons.misc.AbstractCase
+import io.udash.i18n.{LangProperty, TranslationKey, TranslationKey0, TranslationProvider}
 import org.scalajs.dom
 
+import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.{Duration, DurationInt}
 import scala.language.postfixOps
 import scala.scalajs.js
 
-trait TooltipUtils[TooltipType <: Listenable[TooltipType, _]] {
+trait Tooltip[EventType <: ListenableEvent[ThisType], ThisType <: Tooltip[EventType, ThisType]] extends Listenable[ThisType, EventType] {
+  /** Shows the tooltip. */
+  def show(): Unit
+
+  /** Hides the tooltip. */
+  def hide(): Unit
+
+  /** Toggles tooltip visibility. */
+  def toggle(): Unit
+
+  /** Destroys the tooltip. */
```

Comment:
Needs more details. What changes in its behaviour afterwards? Is it important to destroy those?